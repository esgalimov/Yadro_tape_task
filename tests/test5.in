23 4 5 345 3
 12 1 2  1 5
  3 36 4 56 4
   4 64 6 3 52
    42 42  3 34 6
     36 4 64   2 42
      41 41 3 1 4
       23 3 53 6 5
        63 52  2 42 0