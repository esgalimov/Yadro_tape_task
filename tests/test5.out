0 1 1 1 2 2 2 3 3 3 3 3 3 4 4 4 4 4 4 5 5 5 6 6 6 12 23 23 34 36 36 41 41 42 42 42 42 52 52 53 56 63 64 64 345 